includes/class-wc-payments-order-success-page.php

```diff
@@ -0,0 +1,82 @@
+<?php
+/**
+ * Class WC_Payments_Order_Service
```

Comment:
Should be `WC_Payments_Order_Success_Page`

Reply:
Fixed with 592c4a815d7be01ae5b0916ebeb3f9a7d50262bd